### ISSUE TITLE
You may want to turn off the autowatch feature!

### ISSUE THREAD

**Issue by AlexDaniel** [Raku/please-turn-off-the-autowatch-feature#1]
This ticket makes sure that new users receive a notification and get to learn about the autowatch feature and consequences of having it enabled.
  
See https://github.com/Raku/please-turn-off-the-autowatch-feature

**Comment by AlexDaniel** [comment]
Bump!

**Comment by duncand** [comment]
This is good information, but I for one WANT the auto-notifications turned on by default for me.  My email client already filters these into a dedicated folder so they don't bury other stuff, and meanwhile I find out in a timely manner about anything I might be interested in.

**Comment by Scimon** [comment]
Perhaps changing it to "You may want to turn off the autowatch feature" personally I need it for some work stuff.

**Comment by AlexDaniel** [comment]
Sure, if you don't mind getting notifications for ≈150 repos, then it's probably not a problem to ignore this thread.

**Comment by duncand** [comment]
I agree with Scimon though, I feel saying "you may want to" is better wording; saying "please turn off" reads more like an instruction, such as if someone not doing so could cause problems for people besides themselves.

**Comment by AlexDaniel** [comment]
@duncand OK! I changed the title of this ticket, but the repo name will probably have to stay the same, unless we can come up with a short string that is as effective.

**Comment by AlexDaniel** [comment]
I'll be adding everyone to the Raku team soon.
